apps/masterbots.ai/app/actions/ai-executers.ts

```
@@ -0,0 +1,275 @@
+      // ? See: https://rapidapi.com/facundoPri/api/url-to-metadata/playground/apiendpoint_830f2c62-e40f-4b62-9fe0-a5dbc2bc07e6
+      sources
+    }
+  } catch (error) {
+    console.error('Error fetching app data: ', error)
+    return 'Internal Server Error while fetching app data'
+  }
+}
+
+export async function getPromptDetails(promptId: string) {
+  let data = null
+  let error = null
```

Biome

[error] This let declares a variable that is only assigned once.

'error' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)